.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Run dependencies versions update
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
what about other ecosystems, like `docker` or `GHA`...?

// https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

Reply:
In the next step.

Reply:
if we want to do this in the next step, please add a TODO about that in the workflow and fill an issue

Reply:
just an issue is ok, we wouldn't have to wait for the CI to re-run

Reply:
@bratpiorka https://github.com/oneapi-src/unified-memory-framework/issues/539